File: CursoLogica/Linguagens/Python/Atividades/crescente.py
print("Digite dois números: ")

valor1 = int(input())
valor2 = int(input())

while valor1 != valor2:
    if valor1 > valor2:
        print("DECRESCENTE!")
    else:
        print("CRESCENTE!")
    print("Digite outros dois números: ")
    valor1 = int(input())
    valor2 = int(input())

File: CursoLogica/Linguagens/C++/Atividades/crescente.cpp
#include <iomanip>
#include <iostream>

using namespace std;

int main() {

    int valor1, valor2;

    cout << "Digite dois numeros: \n";
    cin >> valor1 >> valor2;

    while (valor1 != valor2) {

        if (valor1 < valor2) {
            cout << "CRESCENTE";
        }
        else if (valor1 > valor2) {
            cout << "DECRESCENTE";
        }

        cout << "\nDigite outros dois numeros: \n";
        cin >> valor1 >> valor2;
    }

    return 0;
}
